src/components/canrawview/canrawview_p.h

```diff
@@ -58,12 +61,15 @@ class CanRawViewPrivate : public QWidget {
 
         QList<QVariant> qvList;
         QList<QStandardItem*> list;
+    
+        int frameID = frame.frameId();
+        QString time = QString::number((double)timer->elapsed() / 1000, 'f', 2);        
 
         qvList.append(rowID++);
-        qvList.append(QString::number((double)timer->elapsed() / 1000, 'f', 2).toDouble());
-        qvList.append(QString::number((double)timer->elapsed() / 1000, 'f', 2));
-        qvList.append(frame.frameId());
-        qvList.append(QString("0x" + QString::number(frame.frameId(), 16)));
+        qvList.append(time.toDouble());
```

Comment:
`time` is a string created from floating point `timer->elapsed()`, and here we again go back to `double`. Can we make a variable of type double that contains time, a then create once a string with a that time value? If so, we can limit number of conversions and move the resulting string over.